### PR TITLE
Fixing test failure for flexible app for Symfony

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: appengine/flexible/symfony/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # This file will go away once gcloud implements fingerprinting.
```
Comment (tmatsuo) [inline appengine/flexible/symfony/Dockerfile]: TODO: use `runtime: php` and remove `Dockerfile`, filed #148

File: appengine/flexible/symfony/php.ini [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-; This file can be used to customize the php configuration for your
-; Managed VMs instances. The default PHP runtime includes the suhosin extension.
-; Functions can be whitelisted to keep the blacklist unmodified.
-
-; Below is the list of suhosin blacklisted functions by default. This can be
-; modified by removing functions from the list.
-
-; removed from this list is "libxml_disable_entity_loader" for symfony
-
-suhosin.executor.func.blacklist = escapeshellarg, escapeshellcmd, exec, highlight_file, lchgrp,lchown, link, symlink, passthru, pclose, popen, proc_close, prog_get_status, proc_nice, proc_open, proc_terminate,shell_exec, show_source, system, gc_collect_cycles, gc_enable, gc_disable, gc_enabled, getmypid, getmyuid, getmygid,getrusage, getmyinode, get_current_user, parse_str, phpinfo, phpversion, php_uname
```
Comment (tmatsuo) [inline appengine/flexible/symfony/php.ini]: Actually this file blocked composer

**Comment by tmatsuo** [comment]
Tests are green, and also on our jenkins, the test in question passed.

**Comment by bshaffer** [comment]
LGTM! nice work

**Pull request merged**